integrations/prodigy/ner_fashion_brands/test_project_ner_fashion_brands.py

```diff
@@ -0,0 +1,10 @@
+from spacy.cli.project.run import project_run
+from spacy.cli.project.assets import project_assets
+from pathlib import Path
+
+
+def test_ner_fashion_project():
```

Comment:
The test needs a unique name here.

Reply:
I guess in general I'd also rename the project, rather than having two projects named the same but sitting in a different folder. Do we need the subfolder `prodigy` or can we just have this project in `integrations/prodigy_ner_fashion/` for instance?